src/Core/Worker.php

```diff
@@ -1121,4 +1140,30 @@ public static function endProcess()
 	{
 		return Process::deleteByPid();
 	}
+
+	private static function checkIPC()
+	{
+		dba::e("CREATE TABLE IF NOT EXISTS `worker-ipc` (`key` integer, `jobs` boolean) ENGINE = MEMORY;");
```

Comment:
Shouldn't this be in DBStructure.php?

Reply:
We weren't able to create memory tables. Now we are.